framework/tests/all-systems/java8/memberref/MemberReferences.java

```diff
@@ -112,6 +112,7 @@ void context(Bound bound) {
     }
 }
 
+@SuppressWarnings("returnsreceiver:methodref.receiver.bound.invalid")
```

Comment:
Please include `// See Issue #2931` at the end of the line.  Also can you move the suppression just before the specific line causing the problem?  You should be able to write a `@SuppressWarnings` annotation just before a method declaration